packages/components/src/components/Menu/Menu.tsx

```diff
@@ -15,14 +16,18 @@ export type MenuItemProps = {
   disabled?: boolean;
 };
 
-export type MenuProps = {
+export interface MenuProps extends Omit<AriakitMenuProps, "store"> {
```

Comment:
maybe if we want to avoid using an interface here just because of the `extends`:

```suggestion
export type MenuProps = Omit<AriakitMenuProps, "store"> & {
```